### PR TITLE
Add attribution comments

### LINKED ISSUE #163: Add attribution comments
Add attribution comments in each contract. [Reference](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol#L1-L2).

### PR DESCRIPTION
**Pull request by andrew-fleming**
Resolves #163.

### CODE REVIEW

**Comment by martriay** [comment]
Looks good! Except we need to update to match the new dir structure/naming.

**Comment by martriay** [comment]
I think we can rebump this PR now

**Comment by andrew-fleming** [comment]
This PR also proposes to include an `interfaces` directory in `account`.

File: openzeppelin/account/interfaces/IAccount.cairo [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Cairo Contracts v0.1.0 (account/interfaces/IAccount.cairo)
```
Comment (martriay) [inline openzeppelin/account/interfaces/IAccount.cairo:2]: I don't think we need an interface directory when there's only one interface in there, the I prefix should be enough for most cases.

Comment (martriay) [inline openzeppelin/account/interfaces/IAccount.cairo:2]: I don't think we need an interface directory when there's only one interface in there, the I prefix should be enough for most cases.

File: openzeppelin/introspection/interfaces/IERC165.cairo [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Cairo Contracts v0.1.0 (introspection/interfaces/IERC165.cairo)
```
Comment (martriay) [inline openzeppelin/introspection/interfaces/IERC165.cairo:2]: This `interfaces/` dir could also be avoided.

**Review by martriay — APPROVED** [review]
Looking good, left minor comments!

**Pull request merged**